formulas/wick-base/functions/wick-service-make-override-systemd

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Internal: Creates the override file if it does not already exist.  Also
+# puts it in a state usable by wickServiceAddDependencySystemd.
+#
+# $1 - The name of the service you wish to add a dependency to.
+#
+wickServiceMakeOverrideSystemd() {
+    local service serviceDir overridePath paths
+    wickGetArgument service 0 "$@"
+
+    wickServiceOverridePathSystemd paths "$service"
+
+    serviceDir="${paths[0]}"
+    wickArrayJoin overridePath "${paths[@]}"
+
+    if [[ ! -d "$serviceDir" ]]; then
+        wickDebug "Creating service directory: $serviceDir"
+        # This copies the permissions that systemctl creates itself
+        # when using systemctl edit <service>
+        wickMakeDir --owner=root:root --mode=755  "$serviceDir"
+    fi
+
+    if [[ ! -f "$overridePath" ]]; then
+        wickDebug "Creating override file: $overridePath"
+        touch "$overridePath"
+        # rw for root, r only for the rest
+        chmod 644 "$overridePath"
+    fi
+
+    if ! grep "^[Unit]" "$overridePath"; then
```

Comment:
Grep can write to stdout.  You probably want to use `grep -q` and `2>/dev/null` in case there are issues.